ambry-messageformat/src/test/java/com.github.ambry.messageformat/BlobStoreRecoveryTest.java

```diff
@@ -112,7 +115,17 @@ public StoreKey getStoreKey(DataInputStream value) throws IOException {
   }
 }
 
+@RunWith(Parameterized.class)
 public class BlobStoreRecoveryTest {
+  @Parameterized.Parameters
+  public static List<Object[]> data() {
+    return Arrays.asList(
+        new Object[][]{{MessageFormatRecord.Message_Header_Version_V1}, {MessageFormatRecord.Message_Header_Version_V1}});
+  }
+
+  public BlobStoreRecoveryTest(short headerVersionToUse) {
```

Comment:
same question as hardDeletes. do we have a test here for null encryptionKey in headerV2